index.html

```diff
@@ -60,6 +61,181 @@ <h2 class="heading heading2">I’m a software developer</h2>
+        </div>
+
+        <!-- Project 4 -->
+
+        <div class="works_card">
+          <div class="card_background">
+            <h3 class="card_heading">Multi-Post Stories Gain+Glory</h3>
+            <ul class="lang_container">
+              <li class="language_btn">Ruby On Rails</li>
+              <li class="language_btn">html</li>
+              <li class="language_btn">css</li>
+              <li class="language_btn">JavScript</li>
+            </ul>
+            <a href="#"><button class="button">See Project</button></a>
+          </div>
+        </div>
+
+        <!-- Project 5 -->
+
+        <div class="works_card">
+          <div class="card_background">
+            <h3 class="card_heading">Multi-Post Stories Gain+Glory</h3>
+            <ul class="lang_container">
+              <li class="language_btn">Ruby On Rails</li>
+              <li class="language_btn">html</li>
+              <li class="language_btn">css</li>
+              <li class="language_btn">JavScript</li>
+            </ul>
+            <a href="#"><button class="button">See Project</button></a>
+          </div>
+        </div>
+
+        <!-- Project 6 -->
+
+        <div class="works_card">
+          <div class="card_background">
+            <h3 class="card_heading">Multi-Post Stories Gain+Glory</h3>
+            <ul class="lang_container">
+              <li class="language_btn">Ruby On Rails</li>
+              <li class="language_btn">html</li>
+              <li class="language_btn">css</li>
+              <li class="language_btn">JavScript</li>
+            </ul>
+            <a href="#"><button class="button">See Project</button></a>
+          </div>
+        </div>
+      </section>
+
+      <!-- About Me -->
+
+      <section id="aboutMe">
+        <section class="intro">
+          <h2 class="heading">About me</h2>
+          <p class="subtext">
+            I can help you build a product, feature or website. Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+          <a href="#"><button class="button button_aboutMe">Get My Resume</button></a>
+        </section>
+      </section>
+
+      <!-- Skills -->
+      <section id="skills">
+        <!-- Languages -->
+        <ul class="skill_container">
+          <li class="skills_card">
+            <div class="img_background">
+              <img
+                class="img_skills"
+                src="images/icon-languages.svg"
+                alt="languages_icon"
+              />
+            </div>
+            <h3 class="card_heading">Languages</h3>
+            <div class="languages_skills">
+              <span class="language_btn_skills">Ruby</span>
+              <span class="language_btn_skills">html</span>
+              <span class="language_btn_skills">css</span>
+              <span class="language_btn_skills">JavScript</span>
+            </div>
```

Comment:
- [ ] [As requested by the first reviewer ](https://github.com/Anjacodes/Portfolio/pull/2#discussion_r733863529) Kindly replace the div with the HTML semantic  `ul` tag because it's a list. wrap each technology in a `li`. Kindly make the change in all the categories